src/Github/Api/V3/CreatePullRequestThroughApiCall.php

```diff
@@ -63,16 +62,14 @@ public function __invoke(
 
         $response = $this->client->sendRequest($request);
 
+        Psl\invariant($response->getStatusCode() >= 200 || $response->getStatusCode() <= 299, 'Failed to create pull request through GitHub API.');
```

Comment:
It should be `&&` here

/cc @Ocramius

Reply:
Ouch, will revert these, thanks for noticing!